teamA/jeongDoYoung/app.js

```diff
@@ -18,20 +18,35 @@ const myDataSource = new DataSource({
   port: process.env.TYPEORM_PORT,
   username: process.env.TYPEORM_USERNAME,
   password: process.env.TYPEORM_PASSWORD,
-  database: process.env.TYPEORM_DATABASE,
+  database: process.env.TYPEORM_DATABASE
 });
 
 myDataSource.initialize().then(() => {
   console.log("data source has been initialized");
 });
 
+app.use(express.json());
 app.use(cors());
 app.use(morgan("dev"));
 
-app.get("/ping", (req, res, next) => {
+app.get("/ping", (req, res) => {
   res.json({ message: "pong" });
 });
 
+app.post('/signup', async (req, res) => {
+  const {name,email,profile_image,password} = req.body
```

Comment:
가독성을 위해 다음과 같이 작성할 수 있습니다.

```suggestion
  const { name, email, profile_image, password } = req.body
```